Looks like tests need updating as well

```
golangci-lint run -E goimports,sqlclosecheck,bodyclose,asciicheck,misspell,errorlint --timeout 5m -e "errors.As" -e "errors.Is" ./...
op-node/p2p/store/extended.go:1: : # github.com/ethereum-optimism/optimism/op-node/p2p/store [github.com/ethereum-optimism/optimism/op-node/p2p/store.test]
op-node/p2p/store/serialize_test.go:14:15: cannot use 1923841 (untyped int constant) as "sync/atomic".Int64 value in struct literal
op-node/p2p/store/serialize_test.go:50:17: cannot use 1923841 (untyped int constant) as "sync/atomic".Int64 value in struct literal
op-node/p2p/store/serialize_test.go:74:17: cannot use 1923841 (untyped int constant) as "sync/atomic".Int64 value in struct literal (typecheck)
package store
make: *** [Makefile:13: lint-go] Error 1

Exited with code exit status 2

```

--- a/op-node/p2p/store/scorebook.go
+++ b/op-node/p2p/store/scorebook.go
@@ -2,6 +2,7 @@ package store
 
 import (
 	"context"
+	"sync/atomic"
 	"time"
 
 	"github.com/ethereum-optimism/optimism/op-service/clock"
@@ -17,17 +18,18 @@ const (
 
 var scoresBase = ds.NewKey("/peers/scores")
 
+// LastUpdate requires atomic update operations. Use the helper functions SetLastUpdated and LastUpdated to modify and access this field.
 type scoreRecord struct {
-	PeerScores PeerScores `json:"peerScores"`
 	LastUpdate int64      `json:"lastUpdate"` // unix timestamp in seconds
+	PeerScores PeerScores `json:"peerScores"`
 }
 
 func (s *scoreRecord) SetLastUpdated(t time.Time) {
-	s.LastUpdate = t.Unix()
+	atomic.StoreInt64(&s.LastUpdate, t.Unix())
 }
 
 func (s *scoreRecord) LastUpdated() time.Time {
-	return time.Unix(s.LastUpdate, 0)
+	return time.Unix(atomic.LoadInt64(&s.LastUpdate), 0)
 }
 
 func (s *scoreRecord) MarshalBinary() (data []byte, err error) {

